examples/pytorch/grace/model.py

```diff
@@ -0,0 +1,108 @@
+import torch as th
+import torch.nn as nn
+import torch.nn.functional as F
+from dgl.nn import GraphConv
+
+
+# Multi-layer Graph Convolutional Networks
+class GCN(nn.Module):
+    def __init__(self, in_dim, out_dim, act_fn, num_layers = 2):
+        super(GCN, self).__init__()
+
+        assert num_layers >= 2
+        self.num_layers = num_layers
+        self.convs = nn.ModuleList()
+
+        self.convs.append(GraphConv(in_dim, out_dim * 2))
+        for _ in range(self.num_layers - 2):
```

Comment:
This does not come into effect at all. I guess we can remove the argument and the related code.

Reply:
I guess users might try a different number of layers, so we can just keep it.

Reply:
Fine